CONTRIBUTING.md

```diff
@@ -3,17 +3,17 @@ Your interest in contributing to this project is greatly appreciated. Please kno
 **Here are some ways you can help that doesn't involve coding:**
 
 - Review any active PRs. If the PR needs any changes, please exlpain them in a simple way.
```

Comment:
`sed -i s/exlpain/explain/g`

Reply:
lol not sure how I missed this second time :D